pkgs/top-level/all-packages.nix

```diff
@@ -26027,6 +26027,8 @@ with pkgs;
 
   timescaledb-tune = callPackage ../development/tools/database/timescaledb-tune { };
 
+  libpq = callPackage ../servers/sql/postgresql/libpq.nix { postgresql = postgresql_15; };
```

Comment:
This should always point to the latest release of postgresql. Maybe leave a comment? Or maybe we need a `postgresql_latest` alias?